blocks/nautilus/ff_container/ff_container-dashboard-section/ff_container-dashboard-section.less

```diff
@@ -20,8 +17,9 @@
         font-size: 21rem/@ff_base;
     }
     &__main {
-        #ff_mix-sections > .main();
-        background-color: white;
+       #ff_mix-sections > .main();
+       padding: 0 @ff_size_control_horizontal;
```

Comment:
Indeed!

@roobottom @dkhuntrods note this only fixes Nautilus. Would need doing for Melody (and Folio?) also. Would it make sense to put this into core?